homeassistant/components/meraki/device_tracker.py

```diff
@@ -111,6 +112,7 @@ def _handle(self, hass, data):
                 gps_location = (lat, lng)
 
             attrs = {}
+            attrs["name"] = i.get("name", "None")
```

Comment:
Don't use `"None"` as default. Use `None`.
```suggestion
            attrs["name"] = i.get("name")
```

Reply:
We shouldn't set name both in `attrs` and as the name parameter.

Reply:
How is the name parameter set by the api? Is it always included in the requests